src/scheduler/worker.rs

```diff
@@ -350,7 +350,10 @@ impl<VM: VMBinding> GCWorker<VM> {
         self.tls = tls;
         self.copy = crate::plan::create_gc_worker_context(tls, mmtk);
         loop {
+            probe!(mmtk, work_poll);
```

Comment:
I was curious when I used the code: why use `work_poll` as a signal for work end. You can just have a work_end tracepoint after `do_work_with_stat`.

Reply:
Explained in the comment